http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -119,7 +119,7 @@ paths:
           schema:
             type: array
             items:
-              $ref: '#/definitions/Cluster'
+              $ref: '#/definitions/ListClusterResponse'
```

Comment:
There's another `listCluster` API to change -- the one that's like `/api/clusters/{googleProject}`